aiidalab_widgets_base/__init__.py

```diff
@@ -34,7 +34,7 @@
     StructureManagerWidget,
     StructureUploadWidget,
 )
-from .viewers import register_viewer_widget, viewer
+from .viewers import AiidaNodeViewWidget, register_viewer_widget, viewer  # noqa
```

Comment:
Why the noqa?

Reply:
I just received a random complaint from the pre-commit: 
<img width="891" alt="Screenshot 2021-07-19 at 11 25 07" src="https://user-images.githubusercontent.com/13118485/126137315-3653efc8-136a-4873-8dd6-82eb8785891f.png">

So this is just to silence it. 

Not sure why it wasn't complaining about other stuff, though.

Reply:
You forgot to add it to the `__all__` list. I've pushed a fix.

Reply:
ah, right, sorry :)